3-setup-ingress-ambassador/README.md

```diff
@@ -605,3 +622,54 @@ For more information about performance tuning please visit the [AES Performance
 `Monitoring` and `Logging` is a very important aspect of every `production ready` system. In the sections to follow, you will learn how to enable it via `Prometheus` and `Loki` for the `AES` stack as well as other backend services.
 
 Go to [Section 4 - Set up Prometheus Stack](../4-setup-prometheus-stack)
+
+### Performance Improvement for the AES 
```

Comment:
Let us not make any change to Ambassador tutorial anymore.

The following section already contains how to reduce replicas to 2 and requests to 200M.
https://github.com/digitalocean/Kubernetes-Starter-Kit-Developers/tree/main/3-setup-ingress-ambassador#performance-considerations-for-the-aes

We should not change the limits, as the Ambassador team have done a lot of testing to arrive at that value.

Can you please remove any reference to Ambassador chapter from your PR?